models/biocro/R/run.biocro.R

```diff
@@ -138,6 +139,8 @@ run.biocro <- function(lat, lon, metpath, soil.nc = NULL, config = config, coppi
   daily.results <- hourly.results[, list(Stem = max(Stem), 
                                          Leaf = max(Leaf),
                                          Root = max(Root), 
+                                         AboveLitter = max(AboveLitter),
```

Comment:
Sorry I read the code too quickly and thought this was hourly not daily. This looks ok. do h2o quantities have time in the denominator (fluxes) or would sum work for sub hourly data?

If easiest route is to require hourly time steps then please add an error if either met input or model output are not hourly.